### ISSUE TITLE
Load dragula from NPM instead of Bower

### ISSUE THREAD

**Issue by oskarrough** [kalcifer/ember-dragula#29]
As bower is being phased out of ember-cli we should move the dragula dependency to NPM. And make sure it is not loaded in Fastboot environments.

**Comment by kkincade** [comment]
Is there any plan to do this in the near future? Looks like the PR is ready.

**Comment by oskarrough** [comment]
@kkincade yes I started it here https://github.com/kalcifer/ember-dragula/pull/30.

But I'm honestly not sure if I missed something.  Ah, I merged it to master now to get the ball moving. Do you have the chance to test it out from master? Before we push a release, that is.

**Comment by ro0gr** [comment]
@oskarrough I think it [doesn't](https://github.com/kalcifer/ember-dragula/pull/30/files#r171206214. ) work currently.

**Comment by ro0gr** [comment]
@oskarrough would you accept a pr fixing this?

**Comment by oskarrough** [comment]
@ro0gr definitely. A PR would be very much appreciated.
